[DOC-11986] Rework the Student Record Tutorial for the Server docs (#167)
couchbaselabs/docs-devex: modules/tutorials/pages/tutorial-install-server.adoc

Reviewer: It's great that you've implemented a page alias, so that users going to `install-couchbase-server` will get redirected to this page automatically. But do you actually need to rename the file? The file hasn't moved to a different module or component, so I would suggest leaving this file with the original name and removing the alias. I don't think it matters if the filename doesn't match the title.

> = Install the Couchbase Server

Reviewer 2: ```suggestion
= Install Couchbase Server
```

> :description: Install and set up the Couchbase Server to continue following the Student Record System tutorial.

Reviewer 2: ```suggestion
:description: Install and set up Couchbase Server to continue following the Student Record System tutorial.
```

:page-topic-type: tutorial
:imagesdir: ../images
:page-pagination: full
:page-toclevels: 2
:page-aliases: install-couchbase-server.adoc

:sdk-footnote: footnote:[Software Development Kit]

[abstract]
{description}

== Installation

> You can install and run the Couchbase Server as a standalone application or as a Docker instance.

Reviewer 2: ```suggestion
You can install and run Couchbase Server as a standalone application or as a Docker instance.
```

=== Standalone

> To download the Couchbase Server as a standalone application:

Reviewer 2: ```suggestion
To download Couchbase Server as a standalone application:
```

. Go to the https://www.couchbase.com/downloads/[Couchbase download page].

> . Go to menu:Server[Couchbase Server Community].

Reviewer: Just wondering why Community Edition when the Docker instructions use Enterprise?

Author: Updated the docker instructions to use the community version!

. Select a **Version** and an **OS** and click btn:[Download].

> After the Couchbase Server is finished downloading, follow the instructions to install it on your machine.

Reviewer 2: ```suggestion
After Couchbase Server is finished downloading, follow the instructions to install it on your machine.
```

=== Docker

> To download the Couchbase Server as a Docker instance:

Reviewer 2: ```suggestion
To download Couchbase Server as a Docker instance:
```

. Download https://www.docker.com/get-started/[Docker^].
. Open a terminal window and run this command to provision the server image, set up the storage location, and run the server:
+
[source, sh]
----
docker run -t --name db -p 8091-8096:8091-8096 -p 11210-11211:11210-11211 \

> couchbase/server:enterprise-{current-enterprise-version}

Reviewer: The `{current-enterprise-version}` attribute isn't defined, which means this example doesn't work. We need to do one of the following:

1. reinstate `tutorial-globals.adoc` and include that partial in this document, or
2. see if we can use the global `{page-version}` attribute in place of `{current-enterprise-version}`, or
3. just replace `{current-enterprise-version}` with `7.6`.

Number one is the most likely choice I think. Number two would be my favorite, except that the value of `{page-version}` has quotes, for some reason. The last one is more maintenance for future authors and might get missed.

----

== Set Up a Cluster

> A Couchbase Server can run as its own cluster or join another cluster to form a multi-node system.

Reviewer 2: ```suggestion
Couchbase Server can run as its own cluster or join another cluster to form a multi-node system.
```

For this tutorial, your server runs as a single node.

To set up a cluster:

. Go to your server's configuration page.
You can access this page in any of the following ways:
* Opening your local Couchbase Server app.
* Running a Docker instance in the terminal.

> * Going directly to `pass:[http://127.0.0.1:8091]` in your browser.

Reviewer: I'm a bit confused about this because it implies that these are three equal options. To set up a cluster, _as a prerequisite_ you either have to run the local Couchbase Server app, or run Couchbase Server in Docker. Then, whichever method you are using, you then have to go to 127.0.0.1 in your browser.

. Click btn:[Setup New Cluster].
. For your cluster name, enter **student-cluster**.

> . Leave the username as **Administrator** and set the password as **password**.

Reviewer 2: I wouldn't recommend telling people a specific username and password - this could expose security issues. 

Tell them to set a username and password, and in any places you've used those values in the code samples, just replace with placeholder values.

. Click btn:[Next: Accept Terms].
. Accept the terms and conditions and click btn:[Finish With Defaults] to access the **Dashboard** page.

== Next Steps

> After installing Couchbase Server and setting up a cluster, you can xref:tutorial-create-database.adoc[create a cluster].

Reviewer: Confusing wording. The next step is to create the cluster? Hang on a second, I thought I already had created the cluster?

```suggestion
After installing Couchbase Server and creating a cluster, you can xref:tutorial-create-database.adoc[implement the database design].
```
